### PR TITLE
Added Telerik author with Xamarin-only article filter.

### PR DESCRIPTION
**Pull request by LanceMcCarthy**
:pencil2: **Blog Url**: 
https://www.telerik.com/blogs/

:scroll: **Feed Url**: 
https://feeds.telerik.com/blogs/

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apparent that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [x] I have applied a filter (Optional: if blog is only about Xamarin or you use `xamarin` in the title or categories in your blog posts)
- [x] I have added my Author class to the csproj file
- [x] I have implemented `IAmACommunityMember` or one of the other derivatives
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

File: src/Firehose.Web/Authors/Telerik.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+		// Bedford, MA Headquarters.
+		public GeoPosition Position => new GeoPosition(42.512778, -71.251500); 
+
+		public string FeedLanguageCode => "en";
+
+		// Categories are not suffient enough for filer. Identifying 'xamarin' in the Title will be sufficient.
+		public bool Filter(SyndicationItem item)
+		{
+			return item.Title.Text.ToLowerInvariant().Contains("xamarin");
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/Telerik.cs]: We already filter Xamarin in title and category. So please go ahead and remove the `IFilterMyBlogPosts` interface and method.

**Comment by LanceMcCarthy** [comment]
Thanks Tomasz, let me know if there's anything else you need.

**Comment by Cheesebaron** [comment]
https://feeds.telerik.com/blogs/ seems to give me a 404 removing the trailing `/` seems to fix it.

File: src/Firehose.Web/Authors/Telerik.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+		public string GitHubHandle => "telerik";
+
+		public string GravatarHash => string.Empty;
+
+		public IEnumerable<Uri> FeedUris
+		{
+			get
+			{
+				yield return new Uri("https://feeds.telerik.com/blogs/");
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/Telerik.cs]: Please remove the trailing `/` otherwise the feed returns 404

**Comment by Cheesebaron** [comment]
@LanceMcCarthy sorry it has taken this long. We've just added back our integration tests, and the feed parser seems to fail finding any Xamarin posts in your RSS feed. I checked the feed and it looks like it is right about that.

**Comment by LanceMcCarthy** [comment]
Thanks for the follow up. How is the feed parsed? There were two titles in August with the word "Xamarin" in it. 

Generally speaking the writers will use the Xamarin Tag (https://www.telerik.com/blogs/tag/xamarin), but we've been trying to make sure Xamarin is in the title for newer content.

**Comment by LanceMcCarthy** [comment]
There was a merge conflict because the branch was missing one new author since we last talked. Fixed.

**Comment by Cheesebaron** [comment]
Digging deeper into the parsing of the feed it fails with:

```
'Text' is an invalid XmlNodeType. Line 66, position 123.
```

You can quickly test that with a simple piece of code like:

```csharp
HttpResponseMessage response;
try
{
    var client = new HttpClient();
    response = await client.GetAsync("https://feeds.telerik.com/blogs").ConfigureAwait(false);
    if (response.IsSuccessStatusCode)
    {
        using (var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false))
        using (var reader = new StreamReader(stream))
        using (var xmlReader = XmlReader.Create(reader))
        {
            var feed = SyndicationFeed.Load(xmlReader);
        }
    }
}
catch(Exception ex)
{
}
```

**Comment by LanceMcCarthy** [comment]
I'm indeed seeing a few XmlParseExceptions now. I tried using the FeedsController to catch where this happens, but I'm guessing that's too late. Where else can I just throw in a breakpoint to see the details of the parsing for the feed URL?

As far as the Feed's Atom XML goes, it look's pretty straightforward:

image

I'll dig deeper to see if I can figure out where it's failing. Thannks for taking a look.

**Comment by LanceMcCarthy** [comment]
@Cheesebaron @jamesmontemagno 

I've come back to revisit this as the feed issue has been resolved. 

### Proof 

Here's validation of the feed of the Telerik Feed:

image

### Why Unit Test Fail?

I looked the DevOps output and it failed because of 3 other sources:

image

**Comment by Cheesebaron** [comment]
Cool Looks good to me

**Comment by LanceMcCarthy** [comment]
Tomaz is there anything else you need from me here? I want to make sure I didn't let anything you need from me slip by and delay it again :)

**Comment by Cheesebaron** [comment]
@LanceMcCarthy no waiting for @jamesmontemagno to perhaps explain why there is an on-hold label 😄

**Comment by jamesmontemagno** [comment]
Oh, looks good my fault. Was trying to figure out CI/CD

**Pull request merged**